### ISSUE TITLE
Shared installation not recognized anymore after update --- installation unusable

### ISSUE THREAD

**Issue by DigNative** [MiKTeX/miktex#868]
- [x] I have read and acknowledged this HOWTO article:  
  [Submitting a MiKTeX problem report](https://miktex.org/howto/bug-report)

Updating a shared installation of MiKTeX on Windows with the currently available official package versions leads to a corruption of the entire MiKTeX installation. I tried with five different computers with different installation dates (ranging from February 2020 to January 2021), and the issue is reproducible on all of the computers.

**What I did:**
1. Generate a `initexmf` report to confirm that I have a shared installation on the computer.
```
>initexmf --report
initexmf: security risk: running with elevated privileges
ReportDate: 2021-07-09 14:34:52
CurrentVersion: 20.12
SetupDate: 2021-01-06 15:14:00
SetupVersion: 20.12
Configuration: Regular
GitInfo: 7017ca6 / 2020-12-13 20:26:45
OS: Windows 10 Enterprise, 64-bit, build 17763
SharedSetup: yes
LinkTargetDirectory: c:\apps\MiKTeX\miktex\bin\x64
PathOkay: yes
LastUpdateCheckAdmin: 2021-01-06 15:14:46
LastUpdateAdmin: not yet
LastUpdateDbAdmin: 2021-01-06 15:14:36
SystemAdmin: yes
RootPrivileges: yes
AdminMode: no
Root0: C:\Users\AdminUser\AppData\Roaming\MiKTeX
Root1: C:\Users\AdminUser\AppData\Local\MiKTeX
Root2: C:\ProgramData\MiKTeX
Root3: c:\apps\MiKTeX\
UserInstall: C:\Users\AdminUser\AppData\Roaming\MiKTeX
UserConfig: C:\Users\AdminUser\AppData\Roaming\MiKTeX
UserData: C:\Users\AdminUser\AppData\Local\MiKTeX
CommonInstall: c:\apps\MiKTeX\
CommonConfig: C:\ProgramData\MiKTeX
CommonData: C:\ProgramData\MiKTeX

The following issues were detected:
  1: minor issue: It has been a long time since a MiKTeX administrator has checked for updates.
``` 
2. Launch MiKTeX Console with administrator privileges and switch to MiKTeX administrator mode.
2. Go to the "Updates" pane and perform an update search (without installing the updates).
3. Navigate to the "Packages" pane, reverse sort all packages by "Installed on" column to identify packages not installed. Select all newly available packages and install them.
_I stopped here on the other computers and closed the MiKTeX Console to check if compilation of test documents still works and if I can observe any other issues. This was not the case, so the problematic step is the following._
4. Go back to the "Updates" pane, perform an update search again and install the updates.

**What happened then:**
After installation of all packages is completed and the post-update tasks are performed, MiKTeX is throwing an error:
![screen_2021-07-09_14_20_08](https://user-images.githubusercontent.com/1613087/125085363-ca7ae780-e0ca-11eb-90f4-f16595d21300.png)
```
GENERAL MIKTEX INFORMATION
ReportDate: 2021-07-09 14:56:44
CurrentVersion: 20.12
SetupDate: 2021-01-06 15:14:00
SetupVersion: 20.12
Configuration: Regular
GitInfo: 7017ca6 / 2020-12-13 20:26:45
OS: Windows 10 Enterprise, 64-bit, build 17763
SharedSetup: yes
LinkTargetDirectory: C:\apps\MiKTeX\miktex\bin\x64
PathOkay: yes
LastUpdateCheckAdmin: 2021-07-09 14:46:05
LastUpdateAdmin: 2021-07-09 14:51:44
LastUpdateDbAdmin: 2021-07-09 14:46:03
SystemAdmin: yes
RootPrivileges: yes
AdminMode: yes
Root0: C:\ProgramData\MiKTeX
Root1: c:\apps\MiKTeX\
CommonInstall: c:\apps\MiKTeX\
CommonConfig: C:\ProgramData\MiKTeX
CommonData: C:\ProgramData\MiKTeX
Invokers: 

ERROR DETAILS
Program: initexmf.exe
Error: The executed process did not succeed.
Details: 
  fileName: c:\apps\MiKTeX\miktex\bin\x64\initexmf.exe
  exitCode: 1
```

In the `C:\ProgramData\MiKTeX\miktex\log\miktex-console_admin.log` file, I can see that actually `initexmf --admin --mkmaps --mklinks` is being executed by the MiKTeX Console:

```
2021-07-09 14:51:45,767+0200 INFO  miktex-console.core - start process: initexmf --admin --mkmaps --mklinks
2021-07-09 14:51:47,486+0200 ERROR mainwindow - :0: The executed process did not succeed.(fileName="c:\apps\MiKTeX\miktex\bin\x64\initexmf.exe", exitCode="1")
2021-07-09 14:57:02,078+0200 WARN  main - session.use_count() == 8
2021-07-09 14:57:02,078+0200 INFO  main - finishing with exit code 0
```

Running this command in an elevated command prompt gives me the following error:
```
initexmf --admin --mkmaps --mklinks
Option --admin only makes sense for a shared MiKTeX setup.

Sorry, but "MiKTeX Configuration Utility" did not succeed for the following reason:

  Option --admin only makes sense for a shared MiKTeX setup.
```

Checking the `initexmf` report confirms that the setup is now deemed as private:
```
>initexmf --report
initexmf: security risk: running with elevated privileges
187 [0x00001d4c] INFO initexmf null - this is MiKTeX Configuration Utility 4.4.1 (MiKTeX 21.6.28)
187 [0x00001d4c] INFO initexmf null - this process (9652) started by 'cmd' with command line: initexmf --report
187 [0x00001d4c] INFO initexmf null - Operating on the private (per-user) MiKTeX setup
ReportDate: 2021-07-09 15:21:40
CurrentVersion: 21.6.28
SetupDate: not yet
SetupVersion: 20.12
Configuration: Regular
GitInfo: b96a3a3 / 2021-06-26 19:20:18
OS: Windows 10 Enterprise, 64-bit, build 17763
SharedSetup: no
LinkTargetDirectory: c:\apps\MiKTeX\miktex\bin\x64
PathOkay: yes
SystemAdmin: yes
RootPrivileges: yes
AdminMode: no
Root0: C:\Users\AdminUser\AppData\Roaming\MiKTeX
Root1: C:\Users\AdminUser\AppData\Local\MiKTeX
UserInstall: C:\Users\AdminUser\AppData\Roaming\MiKTeX
UserConfig: C:\Users\AdminUser\AppData\Roaming\MiKTeX
UserData: C:\Users\AdminUser\AppData\Local\MiKTeX

The following issues were detected:
  1: major issue: So far, you have not checked for MiKTeX updates.
312 [0x00001d4c] INFO initexmf null - this process (9652) finishes with exit code 0
```

After this, I am not able to use MiKTeX normally anymore or to bring it back to original shared installation state. The MiKTeX console shows me a user mode installation without any TEXMF trees configured or packages installed:

![screen_2021-07-09_15_46_47](https://user-images.githubusercontent.com/1613087/125087868-27779d00-e0cd-11eb-8f64-169125ee76a5.png)
![screen_2021-07-09_15_47_42](https://user-images.githubusercontent.com/1613087/125087876-2a728d80-e0cd-11eb-88cf-ef642b098982.png)
![screen_2021-07-09_15_47_54](https://user-images.githubusercontent.com/1613087/125087884-2c3c5100-e0cd-11eb-9f31-ff4fefca73de.png)
![screen_2021-07-09_15_48_06](https://user-images.githubusercontent.com/1613087/125087889-2d6d7e00-e0cd-11eb-97e0-69156bb88cdf.png)

Trying to execute `xelatex` or other programs now gives me all kinds of errors, for example:

![screen_2021-07-09_15_50_06](https://user-images.githubusercontent.com/1613087/125088123-69a0de80-e0cd-11eb-8292-2668fc442dba.png)

I found no way of fixing this. In the end, only a complete reinstallation using the current version of the basic installer (`basic-miktex-21.6-x64.exe`) and then installing all other packages helps.

### Additional Background Information

1. Our computers are operated in a corporate environment, where usually more than one user has access to the computer, so having a shared installation is necessary for us to avoid private installations for each user individually.
2. All packages on these computers are installed in admin mode, the installation of packages on the fly is disabled to prevent automatic user mode package installations.
2. Actually, also our automated deployment of MiKTeX stopped working: The installation of the most recent version of all packages with the CLI network installer is also leading to a corrupted installation exhibiting the same issues (no shared installation shown, instead non-functional private installation).

**Comment by edocevoli** [comment]
Thank you, this seems to be a special setup configuration. I cannot reproduce this bug with a regular installation. That said, I cannot work on this issue effectively.

The shared setup flag is stored in these places:

1. registry: `HKLM\SOFTWARE\MiKTeX.org\MiKTeX\Core`: `SharedSetup=1`
2. config file: miktex.ini: `[Core]`: `SharedSetup=1`

Maybe this info helps you to investigate the issue.

The issue will be reopened when we have enough info in order to fix the bug.

**Comment by DigNative** [comment]
Thank you for your very fast answer and the hints for further investigation.

I think the setup is not really special in that sense; I would rather assume you probably have no "old enough" installation at hand which triggers the issue. I also cannot reproduce it if the MiKTeX installation is only some months old --- the update will then work.

The systems are essentially installed using the CLI network installer (`miktexsetup-4.1-x64.zip`) and the following arguments: `--shared=yes --modify-path=yes --use-registry=no --common-install=c:\apps\MiKTeX\ --package-set=complete --local-package-repository=c:\apps\MiKTeX-packages\ --quiet install`. The path `c:\apps\MiKTeX-packages\` is just a 1:1 copy of the `/systems/win32/miktex/tm/packages/` directory on CTAN on one of our internal servers to relieve load from the CTAN servers and speed up installation.

I just checked the flag in the `miktex.ini` file after ending up with the situation as described in my first post: The `SharedSetup` flag is still set to `1`, so this is fine:
```
;;; DO NOT EDIT THIS FILE!


[Core]
LastAdminMaintenance=1625845499
NoRegistry=1
SharedSetup=1

[MPM]
AuthToken=[REDACTED]
AuthTokenNotValidAfter=1609867599
AutoInstall=0
LastAdminUpdate=1625845823
LastAdminUpdateCheck=1625845501
LastAdminUpdateDb=1625845499
LocalRepository=//[REDACTED]/software-dist/tex/systems/win32/miktex/tm/packages
RepositoryType=local

[Setup]
LastAdminDiagnose=1625845502
```

In principle, the registry should not have an effect as far as I understand, since we are using the `--use-registry=no` option during installation. However, when checking the registry with the information you provided, I can see actually that there is a corresponding key, but also a difference: There is no `HKLM\SOFTWARE\MiKTeX.org\MiKTeX\Core` key on the system: I can only find `HKLM\SOFTWARE\MiKTeX.org\MiKTeX\2.9\Core`. Is it just a mistake that the `2.9` in the path is missing in your post or should the version number really not appear?

**Comment by edocevoli** [comment]
Thank you. I will try to reproduce it with the given info.

**Comment by DigNative** [comment]
Thank you for the effort! My recommendation would be to use an installation as of November 2020. In case you would need the original package repository and setup utility versions of this time, you can use the following link: https://ctan.space-pro.be/repo/!svn/bc/1712/systems/win32/miktex/

It points to a SVN server of me, which creates a snapshot of CTAN every day. Revision 1712 is from November 24, 2020.

**Comment by edocevoli** [comment]
During startup, when the configuration is not yet known, `miktex.ini` is also searched in `c:\apps\MiKTeX\miktex\config`. So maybe this is a workaround for the issue:

```
mkdir c:\apps\MiKTeX\miktex
mkdir c:\apps\MiKTeX\miktex\config
echo [Core]> c:\apps\MiKTeX\miktex\config\miktex.ini
echo SharedSetup=1>> c:\apps\MiKTeX\miktex\config\miktex.ini
```

**Comment by DigNative** [comment]
Thank you, I will try this in a couple of minutes. Another thing (I don't know if it is related or not, but probably you can say better than I): I noticed above that our automated deployment of MiKTeX is currently failing as well. I am in parallel debugging this on the same machines and the MiKTeX setup utility (`miktexsetup`) is also throwing an error:

```
>miktexsetup.exe --shared=yes --modify-path=yes --use-registry=no --common-install=c:\apps\MiKTeX\ --package-set=complete --local-package-repository=c:\apps\miktex-packages\ --verbose install
this is MiKTeX Setup Service 4.2 (MiKTeX 21.1)
starting installer...
Loading package database...
starting package maintenance...
installation directory: c:\apps\MiKTeX\
package repository: c:\apps\miktex-packages\
visiting repository c:\apps\miktex-packages\...
repository type: local package repository
loading package repository manifest...
package repository digest: 4855bdc76af4c516884c58e8ae6c8565
going to install 196615 file(s) (4378 package(s))
extracting files from 12many.tar.lzma...
extracting files from a0poster.tar.lzma...
...
...
...
extracting files from zxjatype.tar.lzma...
extracting files from zztex.tar.lzma...
updating package manifests (c:\apps\MiKTeX\miktex/config\package-manifests.ini)...
installed 136 package manifests
visiting repository c:\apps\miktex-packages\...
repository type: local package repository
loading package repository manifest...
turning on administrator mode
MiKTeX encountered an internal error.
Data:
Source: Libraries\MiKTeX\Core\Session\texmfroot.cpp:238
MiKTeX encountered an internal error.
Info:
Source: Libraries\MiKTeX\Core\Session\texmfroot.cpp
Line: 238

Sorry, but "MiKTeX Configuration Utility" did not succeed.
log4cxx: No appender could be found for logger (initexmf).
log4cxx: Please initialize the log4cxx system properly.
initexmf: MiKTeX encountered an internal error.
```

Do you think it could be related or should I better open another issue for it?

**Comment by DigNative** [comment]
I tried your suggestion, but had to add a space in the last command before the `>>`:
> 
> ```
> mkdir c:\apps\MiKTeX\miktex
> mkdir c:\apps\MiKTeX\miktex\config
> echo [Core]> c:\apps\MiKTeX\miktex\config\miktex.ini
> echo SharedSetup=1 >> c:\apps\MiKTeX\miktex\config\miktex.ini
> ```

Indeed, this brought back the shared installation in the MiKTeX Console: I can see now all packages again and the configuration also seems normal as before. However, upon executing `xelatex`, for example, I receive a couple of DLL errors:

![screen_2021-07-09_19_20_23](https://user-images.githubusercontent.com/1613087/125114939-ee4e2580-e0ea-11eb-9edd-5c38655ac474.png)

I can see these files are in the `C:\apps\MiKTeX\miktex\bin\x64` folder, but with a different version number:
<details>
<summary>Directory Listing c:\apps\MiKTeX\miktex\bin\x64</summary>

```
 Directory of c:\apps\MiKTeX\miktex\bin\x64

2021/07/09  19:14    <DIR>          .
2021/07/09  19:14    <DIR>          ..
2020/12/14  11:05            63,488 a2ping.exe
2020/12/14  11:11         6,423,552 a5toa4.exe
2020/12/14  11:05            63,488 abs2rel.exe
2020/12/14  11:05            63,488 afm2afm.exe
2020/12/14  11:00           107,520 afm2tfm.exe
2020/12/14  11:05         2,630,656 amstex.exe
2019/03/18  19:50            12,224 api-ms-win-core-console-l1-1-0.dll
2019/03/18  19:50            12,224 api-ms-win-core-console-l1-2-0.dll
2019/03/18  19:50            11,712 api-ms-win-core-datetime-l1-1-0.dll
2019/03/18  19:50            11,720 api-ms-win-core-debug-l1-1-0.dll
2019/03/18  19:50            11,720 api-ms-win-core-errorhandling-l1-1-0.dll
2019/03/18  19:50            15,304 api-ms-win-core-file-l1-1-0.dll
2019/03/18  19:50            11,712 api-ms-win-core-file-l1-2-0.dll
2019/03/18  19:50            11,720 api-ms-win-core-file-l2-1-0.dll
2019/03/18  19:50            11,720 api-ms-win-core-handle-l1-1-0.dll
2019/03/18  19:50            12,232 api-ms-win-core-heap-l1-1-0.dll
2019/03/18  19:50            11,712 api-ms-win-core-interlocked-l1-1-0.dll
2019/03/18  19:50            12,736 api-ms-win-core-libraryloader-l1-1-0.dll
2019/03/18  19:50            14,280 api-ms-win-core-localization-l1-2-0.dll
2019/03/18  19:50            12,224 api-ms-win-core-memory-l1-1-0.dll
2019/03/18  19:50            11,720 api-ms-win-core-namedpipe-l1-1-0.dll
2019/03/18  19:50            12,744 api-ms-win-core-processenvironment-l1-1-0.dll
2019/03/18  19:50            14,272 api-ms-win-core-processthreads-l1-1-0.dll
2019/03/18  19:50            12,232 api-ms-win-core-processthreads-l1-1-1.dll
2019/03/18  19:50            11,200 api-ms-win-core-profile-l1-1-0.dll
2019/03/18  19:50            12,224 api-ms-win-core-rtlsupport-l1-1-0.dll
2019/03/18  19:50            11,720 api-ms-win-core-string-l1-1-0.dll
2019/03/18  19:50            13,760 api-ms-win-core-synch-l1-1-0.dll
2019/03/18  19:50            12,232 api-ms-win-core-synch-l1-2-0.dll
2019/03/18  19:50            12,744 api-ms-win-core-sysinfo-l1-1-0.dll
2019/03/18  19:50            12,232 api-ms-win-core-timezone-l1-1-0.dll
2019/03/18  19:50            11,720 api-ms-win-core-util-l1-1-0.dll
2019/03/18  19:50            12,744 api-ms-win-crt-conio-l1-1-0.dll
2019/03/18  19:50            15,816 api-ms-win-crt-convert-l1-1-0.dll
2019/03/18  19:50            12,232 api-ms-win-crt-environment-l1-1-0.dll
2019/03/18  19:50            13,768 api-ms-win-crt-filesystem-l1-1-0.dll
2019/03/18  19:50            12,744 api-ms-win-crt-heap-l1-1-0.dll
2019/03/18  19:50            12,232 api-ms-win-crt-locale-l1-1-0.dll
2019/03/18  19:50            20,928 api-ms-win-crt-math-l1-1-0.dll
2019/03/18  19:50            19,904 api-ms-win-crt-multibyte-l1-1-0.dll
2019/03/18  19:50            64,456 api-ms-win-crt-private-l1-1-0.dll
2019/03/18  19:50            12,736 api-ms-win-crt-process-l1-1-0.dll
2019/03/18  19:50            16,328 api-ms-win-crt-runtime-l1-1-0.dll
2019/03/18  19:50            17,864 api-ms-win-crt-stdio-l1-1-0.dll
2019/03/18  19:50            18,368 api-ms-win-crt-string-l1-1-0.dll
2019/03/18  19:50            14,280 api-ms-win-crt-time-l1-1-0.dll
2019/03/18  19:50            12,224 api-ms-win-crt-utility-l1-1-0.dll
2020/12/14  11:05            63,488 arara.exe
2021/06/26  13:03           238,592 arctrl.exe
2020/12/14  11:05            63,488 arlatex.exe
2020/12/14  11:07         5,091,840 asy.exe
2020/12/14  11:05            63,488 authorindex.exe
2020/12/14  11:05            63,488 autoinst.exe
2020/12/14  10:57            81,920 autosp.exe
2020/12/14  10:57           136,192 axohelp.exe
2020/12/14  11:05         2,630,656 bagpipe.exe
2020/12/14  11:05         2,630,656 bagpipex.exe
2020/12/14  10:58            36,864 bg5conv.exe
2020/12/14  10:58           114,688 bg5latex.exe
2020/12/14  10:58           114,688 bg5pdflatex.exe
2020/12/14  10:58           114,688 bg5platex.exe
2020/12/14  10:58           114,688 bg5pluslatex.exe
2020/12/14  10:58           114,688 bg5pluspdflatex.exe
2020/12/14  10:58           114,688 bg5ppdflatex.exe
2020/12/14  11:05            63,488 bib2gls.exe
2020/12/14  11:05            63,488 bib2xhtml.exe
2020/12/14  11:05            63,488 bibdoiadd.exe
2020/12/14  11:05            63,488 biber.exe
2020/12/14  11:05            63,488 bibhtml.exe
2020/12/14  10:57           208,384 bibsort.exe
2020/12/14  10:57           296,960 bibtex.exe
2020/12/14  10:57           212,992 bibtex8.exe
2020/12/14  10:57           219,648 bibtexu.exe
2020/12/14  11:05            63,488 bibzbladd.exe
2020/12/14  11:05            63,488 biokey2html.exe
2020/12/14  11:05            63,488 birm.exe
2020/12/14  11:05            63,488 bmeditor.exe
2020/12/14  11:05         2,630,656 bplain.exe
2020/12/14  11:05            63,488 bundledoc.exe
2020/12/14  10:58            40,448 cef5conv.exe
2020/12/14  10:58           114,688 cef5latex.exe
2020/12/14  10:58           114,688 cef5pdflatex.exe
2020/12/14  10:58            39,936 cefconv.exe
2020/12/14  10:58           114,688 ceflatex.exe
2020/12/14  10:58           114,688 cefpdflatex.exe
2020/12/14  10:58            40,448 cefsconv.exe
2020/12/14  10:58           114,688 cefslatex.exe
2020/12/14  10:58           114,688 cefspdflatex.exe
2020/12/14  11:00           460,288 cfftot1.exe
2020/12/14  11:11         6,423,552 checkcites.exe
2020/12/14  11:05            63,488 chklref.exe
2020/12/14  10:57           136,192 chktex.exe
2020/12/14  11:05            63,488 cjk-gs-integrate.exe
2021/06/26  12:14           116,224 cjklatex.exe
2020/12/14  11:11         6,423,552 cluttex.exe
2020/06/09  09:32           317,208 concrt140.dll
2020/12/14  11:05            63,488 convertgls2bib.exe
2020/12/14  11:05            63,488 creatdtx.exe
2020/12/14  11:05         2,630,656 cslatex.exe
2020/12/14  11:05         2,630,656 csplain.exe
2020/12/14  11:05            63,488 csvtools.exe
2020/12/14  11:05            63,488 ctan-o-mat.exe
2020/12/14  11:11         6,423,552 ctanbib.exe
2020/12/14  10:58           108,032 ctangle.exe
2020/12/14  11:05            63,488 ctanify.exe
2020/12/14  11:05            63,488 ctanupload.exe
2020/12/14  10:58           148,992 cweave.exe
2020/12/14  11:05            63,488 dad-to-utf8.exe
2020/12/14  11:05            63,488 de-macro.exe
2020/12/14  11:05            63,488 depythontex.exe
2020/12/14  10:58            98,816 devnag.exe
2020/12/14  11:05            63,488 dosepsbin.exe
2020/12/14  11:05            63,488 dumphint.exe
2020/12/14  11:05            63,488 dviasm.exe
2020/12/14  11:00           207,360 dvicopy.exe
2020/12/14  11:05            63,488 dviinfox.exe
2020/12/14  11:05         2,630,656 dvilollipop.exe
2020/12/14  11:11         6,448,640 dvilualatex-dev.exe
2020/12/14  11:11         6,413,312 dvilualatex.exe
2020/12/14  11:11         6,413,312 dvilualollipop.exe
2020/12/14  11:11         6,413,312 dviluatex.exe
2020/12/14  11:00           966,144 dvipdfm.exe
2020/12/14  11:00           966,144 dvipdfmx.exe
2020/12/14  11:00            81,920 dvipdft.exe
2020/12/14  11:00           161,280 dvipng.exe
2020/12/14  11:00           326,656 dvips.exe
2020/12/14  11:00         9,471,488 dvisvgm.exe
2020/12/14  11:03           162,304 dvitomp.exe
2020/12/14  11:05           189,440 dvitype.exe
2020/12/14  11:00           966,144 ebb.exe
2020/12/14  11:05            63,488 eps2eps.exe
2020/12/14  11:05            81,920 epsffit.exe
2020/12/14  11:11         6,423,552 epspdf.exe
2020/12/14  11:00           393,216 epstopdf.exe
2020/12/14  11:05         2,630,656 etex.exe
2020/12/14  11:05            63,488 etexdef.exe
2020/12/14  11:05            63,488 exceltex.exe
2020/12/14  10:58            36,864 extconv.exe
2020/12/14  11:00           966,144 extractbb.exe
2020/12/14  11:05            63,488 extractres.exe
2020/12/14  10:54            74,240 fc-cache.exe
2020/12/14  10:54            73,216 fc-cat.exe
2020/12/14  10:55            68,096 fc-conflist.exe
2020/12/14  10:55            71,168 fc-list.exe
2020/12/14  10:54            71,168 fc-match.exe
2020/12/14  10:55            70,656 fc-pattern.exe
2020/12/14  10:55            70,656 fc-query.exe
2020/12/14  10:54            70,144 fc-scan.exe
2020/12/14  10:55            71,168 fc-validate.exe
2020/12/14  11:05            63,488 feynmf.exe
2020/12/14  11:05            63,488 fig4latex.exe
2020/12/14  11:05            63,488 findhyph.exe
2021/06/26  12:16           121,856 findtexmf.exe
2020/12/14  11:05            63,488 font2c.exe
2020/12/14  11:00           140,288 fribidixetex.exe
2020/12/14  11:05            63,488 fullref.exe
2020/12/14  10:58           114,688 gbklatex.exe
2020/12/14  10:58           114,688 gbkpdflatex.exe
2020/12/14  11:11         6,423,552 getmapdl.exe
2020/12/14  11:05            63,488 gfi-run.exe
2020/12/14  11:02           185,344 gftodvi.exe
2020/12/14  11:02           144,384 gftopk.exe
2020/12/14  11:02           175,104 gftype.exe
2020/12/14  11:00           824,832 gregorio.exe
2020/12/14  11:05            63,488 gsbj.exe
2020/12/14  11:05            63,488 gsdj.exe
2020/12/14  11:05            63,488 gsdj500.exe
2021/06/26  12:25           173,568 gsf2pk.exe
2020/12/14  11:05            63,488 gslj.exe
2020/12/14  11:05            63,488 gslp.exe
2020/12/14  11:05            63,488 gsnd.exe
2020/12/14  11:05            63,488 gsndt.exe
2020/12/14  11:05            63,488 gssetgs.exe
2020/12/14  11:05            63,488 gst.exe
2020/12/14  11:05            63,488 gstt.exe
2020/12/14  10:58           104,448 hbf2gf.exe
2020/12/14  11:05            63,488 ht.exe
2020/12/14  11:05            63,488 htcontext.exe
2020/12/14  11:05            63,488 htcopy.exe
2020/12/14  11:05            63,488 htlatex.exe
2020/12/14  11:05            63,488 htmex.exe
2020/12/14  11:05            63,488 htmove.exe
2020/12/14  11:05            63,488 httex.exe
2020/12/14  11:05            63,488 httexi.exe
2020/12/14  11:05            63,488 htxelatex.exe
2020/12/14  11:05            63,488 htxetex.exe
2020/12/14  11:05            63,488 ibyhyph.exe
2017/11/02  14:45        26,824,704 icudt60.dll
2017/11/02  14:44         2,254,336 icuin60.dll
2017/11/02  14:42         1,539,584 icuuc60.dll
2020/12/14  11:02           718,336 inimf.exe
2020/12/14  11:06           600,576 initex.exe
2021/06/26  12:25           576,000 initexmf.exe
2021/07/09  19:06    <DIR>          internal
2020/12/14  11:05            63,488 isi2bibtex.exe
2020/12/14  11:05         2,630,656 jadetex.exe
2020/12/14  11:05            63,488 jamo-normalize.exe
2020/12/14  11:05            63,488 jfmutil.exe
2020/12/14  11:05            63,488 ketcindy.exe
2020/12/14  11:05            63,488 komkindex.exe
2020/12/14  11:00            83,968 kpsewhich.exe
2020/12/14  11:11         6,423,552 l3build.exe
2020/12/14  11:00           126,464 lacheck.exe
2020/12/14  11:05         2,630,656 latex-dev.exe
2020/12/14  11:05            63,488 latex-git-log.exe
2020/12/14  11:05            63,488 latex-papersize.exe
2020/12/14  11:05         2,630,656 latex.exe
2020/12/14  11:05            63,488 latex2man.exe
2020/12/14  11:05            63,488 latex2nemeth.exe
2020/12/14  11:05            63,488 latexdef.exe
2020/12/14  11:05            63,488 latexdiff-fast.exe
2020/12/14  11:05            63,488 latexdiff-so.exe
2020/12/14  11:05            63,488 latexdiff-vc.exe
2020/12/14  11:05            63,488 latexdiff.exe
2020/12/14  11:05            63,488 latexindent.exe
2020/12/14  11:05            63,488 latexmk.exe
2020/12/14  11:05            63,488 latexpand.exe
2020/12/14  11:05            63,488 latexrevise.exe
2020/11/06  07:30         3,385,328 libGLESv2.dll
2020/12/14  11:11         6,423,552 llmk.exe
2020/12/14  11:05         2,630,656 lollipop.exe
2020/12/14  11:05            63,488 lp386.exe
2020/12/14  11:05            63,488 lp386r2.exe
2020/12/14  11:05            63,488 lpgs.exe
2020/12/14  11:05            63,488 lpr2.exe
2020/12/14  11:05            63,488 ltx2crossrefxml.exe
2020/12/14  11:05            63,488 ltximg.exe
2020/12/14  11:11         6,423,552 luacolor.exe
2020/12/14  11:11         6,448,640 luahblatex-dev.exe
2020/12/14  11:11         6,448,640 luahblatex.exe
2020/12/14  11:11         6,448,640 luahbtex.exe
2020/12/14  11:11         6,448,640 lualatex-dev.exe
2020/12/14  11:11         6,448,640 lualatex.exe
2020/12/14  11:05            63,488 lualatexdef.exe
2020/12/14  11:11         6,413,312 lualollipop.exe
2020/12/14  11:11         6,423,552 luamesh.exe
2020/12/14  11:11         6,423,552 luaotfload-tool.exe
2020/12/14  11:11         6,413,312 luatex.exe
2020/12/14  11:05            63,488 luatexdef.exe
2020/12/14  11:11         6,423,552 lwarpmk.exe
2020/12/14  11:11         6,423,552 lyluatex.exe
2020/12/14  11:11         6,423,552 m-tx.exe
2020/12/14  11:11         6,423,552 magicnum.exe
2020/12/14  11:11         6,423,552 make4ht.exe
2020/12/14  11:01           161,280 makebase.exe
2020/12/14  11:05            63,488 makedtx.exe
2020/12/14  11:01           220,160 makefmt.exe
2020/12/14  11:11         6,423,552 makeglossaries-lite.exe
2020/12/14  11:05            63,488 makeglossaries.exe
2020/12/14  11:01           155,136 makeindex.exe
2020/12/14  11:05            63,488 makejmlrbook.exe
2020/12/14  11:01           210,944 makepk.exe
2020/12/14  11:01           186,880 maketfm.exe
2020/12/14  11:05            63,488 mathspic.exe
2020/12/14  11:05            63,488 mathspic113.exe
2020/12/14  11:05            63,488 merge-tex.exe
2020/12/14  11:02           718,336 mf.exe
2020/12/14  11:05            63,488 mf2pt1.exe
2020/06/09  09:32         5,588,256 mfc140.dll
2020/06/09  09:32            47,600 mfc140chs.dll
2020/06/09  09:32            47,624 mfc140cht.dll
2020/06/09  09:32            76,296 mfc140deu.dll
2020/06/09  09:32            66,544 mfc140enu.dll
2020/06/09  09:32            75,040 mfc140esn.dll
2020/06/09  09:32            76,272 mfc140fra.dll
2020/06/09  09:32            74,016 mfc140ita.dll
2020/06/09  09:32            55,792 mfc140jpn.dll
2020/06/09  09:32            54,560 mfc140kor.dll
2020/06/09  09:32            72,176 mfc140rus.dll
2020/06/09  09:32         5,621,024 mfc140u.dll
2020/06/09  09:32            92,680 mfcm140.dll
2020/06/09  09:32            92,656 mfcm140u.dll
2020/12/14  11:05            63,488 mfirstuc.exe
2020/12/14  11:02           162,816 mft.exe
2020/12/14  11:05            63,488 mgetmref.exe
2018/09/25  18:19           222,208 mgs.exe
2018/09/25  18:19        18,396,160 mgsdll64.dll
2021/06/26  12:20           109,568 miktex-afm2tfm.exe
2021/06/26  12:49         5,072,384 miktex-asy.exe
2021/06/26  12:12            83,456 miktex-autosp.exe
2021/06/26  12:12           138,752 miktex-axohelp.exe
2021/06/26  12:14            37,888 miktex-bg5conv.exe
2021/06/26  12:12           209,920 miktex-bibsort.exe
2021/06/26  12:13           305,152 miktex-bibtex.exe
2021/06/26  12:12           215,552 miktex-bibtex8.exe
2021/06/26  12:13           222,720 miktex-bibtexu.exe
2021/06/26  12:14            41,984 miktex-cef5conv.exe
2021/06/26  12:14            41,984 miktex-cefconv.exe
2021/06/26  12:14            41,984 miktex-cefsconv.exe
2021/06/26  12:27           463,872 miktex-cfftot1.exe
2021/06/26  12:13           137,216 miktex-chktex.exe
2021/06/26  13:06         1,017,344 miktex-console.exe
2021/06/26  13:05         1,017,344 miktex-console_admin.exe
2020/12/14  11:08           894,976 miktex-console_admin.exe.(old)
2021/06/26  12:14           109,568 miktex-ctangle.exe
2021/06/26  12:16           151,040 miktex-cweave.exe
2021/06/26  12:16            99,840 miktex-devnag.exe
2021/06/26  12:18           208,384 miktex-dvicopy.exe
2021/06/26  12:19           976,384 miktex-dvipdfmx.exe
2021/06/26  12:19            84,480 miktex-dvipdft.exe
2021/06/26  12:19           162,816 miktex-dvipng.exe
2021/06/26  12:20           331,776 miktex-dvips.exe
2021/06/26  12:25         9,521,152 miktex-dvisvgm.exe
2021/06/26  12:41           190,976 miktex-dvitype.exe
2021/06/26  12:40            83,968 miktex-epsffit.exe
2021/06/26  12:25           462,848 miktex-epstopdf-unrestricted.exe
2021/06/26  12:25           462,848 miktex-epstopdf.exe
2021/06/26  12:14            37,888 miktex-extconv.exe
2021/06/26  11:42            76,800 miktex-fc-cache.exe
2021/06/26  11:42            75,264 miktex-fc-cat.exe
2021/06/26  11:42            71,168 miktex-fc-conflist.exe
2021/06/26  11:42            72,704 miktex-fc-list.exe
2021/06/26  11:41            73,216 miktex-fc-match.exe
2021/06/26  11:42            72,192 miktex-fc-pattern.exe
2021/06/26  11:42            72,704 miktex-fc-query.exe
2021/06/26  11:41            72,192 miktex-fc-scan.exe
2021/06/26  11:42            73,216 miktex-fc-validate.exe
2021/06/26  12:25           141,312 miktex-fribidixetex.exe
2021/06/26  12:33           192,000 miktex-gftodvi.exe
2021/06/26  12:33           146,432 miktex-gftopk.exe
2021/06/26  12:33           178,176 miktex-gftype.exe
2021/06/26  12:25           888,320 miktex-gregorio.exe
2021/06/26  12:14           107,520 miktex-hbf2gf.exe
2021/06/26  12:26            86,016 miktex-kpsewhich.exe
2021/06/26  12:28           128,512 miktex-lacheck.exe
2020/12/14  11:05         2,630,656 miktex-latex.exe
2021/06/26  12:32            48,640 miktex-lua.exe
2021/06/26  12:32            49,664 miktex-luac.exe
2021/06/26  12:31         6,474,240 miktex-luahbtex.exe
2020/12/14  11:11         6,448,640 miktex-lualatex.exe
2021/06/26  12:31         6,438,400 miktex-luatex.exe
2021/06/26  12:32           172,544 miktex-makebase.exe
2021/06/26  12:32           230,400 miktex-makefmt.exe
2021/06/26  12:32           156,672 miktex-makeindex.exe
2021/06/26  12:32           171,008 miktex-makemf.exe
2021/06/26  12:32           219,136 miktex-makepk.exe
2021/06/26  12:32           197,632 miktex-maketfm.exe
2021/06/26  12:34           729,088 miktex-mf.exe
2021/06/26  12:33           164,864 miktex-mft.exe
2021/06/26  12:36            73,728 miktex-mkocp.exe
2021/06/26  12:27           307,200 miktex-mmafm.exe
2021/06/26  12:27           383,488 miktex-mmpfb.exe
2021/06/26  12:35           163,840 miktex-mpost.exe
2021/06/26  12:35           204,800 miktex-odvicopy.exe
2021/06/26  12:35           211,456 miktex-ofm2opl.exe
2021/06/26  12:35           230,400 miktex-opl2ofm.exe
2021/06/26  12:28           579,072 miktex-otfinfo.exe
2021/06/26  12:27           889,344 miktex-otftotfm.exe
2021/06/26  12:36            94,208 miktex-otp2ocp.exe
2021/06/26  12:36            73,216 miktex-outocp.exe
2021/06/26  12:35           236,544 miktex-ovf2ovp.exe
2021/06/26  12:35           251,904 miktex-ovp2ovf.exe
2021/06/26  12:36           167,424 miktex-patgen.exe
2021/06/26  12:05            76,800 miktex-pdfattach.exe
2021/06/26  12:04            83,968 miktex-pdfdetach.exe
2021/06/26  12:04            81,920 miktex-pdffonts.exe
2021/06/26  12:05            98,304 miktex-pdfimages.exe
2021/06/26  12:05           147,968 miktex-pdfinfo.exe
2020/12/14  11:05         2,630,656 miktex-pdflatex.exe
2021/06/26  12:05            78,336 miktex-pdfseparate.exe
2021/06/26  12:38         2,652,160 miktex-pdftex.exe
2021/06/26  12:04           218,112 miktex-pdftocairo.exe
2021/06/26  12:05           204,288 miktex-pdftohtml.exe
2021/06/26  12:04            92,672 miktex-pdftoppm.exe
2021/06/26  12:05            92,160 miktex-pdftops.exe
2021/06/26  12:37         1,528,320 miktex-pdftosrc.exe
2021/06/26  12:04           126,464 miktex-pdftotext.exe
2021/06/26  12:05           104,960 miktex-pdfunite.exe
2021/06/26  12:39            76,288 miktex-pk2bm.exe
2021/06/26  12:41           178,176 miktex-pltotf.exe
2021/06/26  12:39           646,656 miktex-pmxab.exe
2021/06/26  12:41           126,976 miktex-pooltype.exe
2021/06/26  12:32           193,536 miktex-prepmx.exe
2021/06/26  12:40           212,992 miktex-ps2pk.exe
2021/06/26  12:40            81,408 miktex-psbook.exe
2021/06/26  12:40            91,136 miktex-psnup.exe
2021/06/26  12:40            89,088 miktex-psresize.exe
2021/06/26  12:40            82,432 miktex-psselect.exe
2021/06/26  12:40            90,112 miktex-pstops.exe
2021/06/26  12:12            47,616 miktex-rebar.exe
2021/06/26  12:39            65,536 miktex-rungs.exe
2021/06/26  12:39           125,440 miktex-scor2prt.exe
2021/06/26  12:14            37,888 miktex-sjisconv.exe
2021/06/26  12:40           210,432 miktex-synctex.exe
2021/06/26  12:27           379,904 miktex-t1dotlessj.exe
2021/06/26  12:28           331,776 miktex-t1lint.exe
2021/06/26  12:27           336,896 miktex-t1rawafm.exe
2021/06/26  12:28           372,736 miktex-t1reencode.exe
2021/06/26  12:28           352,256 miktex-t1testpage.exe
2021/06/26  12:40           100,352 miktex-t4ht.exe
2021/06/26  12:13           169,984 miktex-tangle.exe
2020/12/14  11:08           894,976 miktex-taskbar-icon.exe
2021/06/26  12:42           612,352 miktex-tex.exe
2021/06/26  12:12            48,128 miktex-tex2aspc.exe
2021/06/26  12:41           177,152 miktex-tex4ht.exe
2020/12/14  11:11         6,413,312 miktex-texlua.exe
2020/12/14  11:11         6,413,312 miktex-texluac.exe
2021/06/26  12:54         7,665,664 miktex-texworks.exe
2021/06/26  12:41           180,736 miktex-tftopl.exe
2021/06/26  12:37           105,984 miktex-ttf2afm.exe
2021/06/26  12:43           158,208 miktex-ttf2pk.exe
2021/06/26  12:43           182,272 miktex-ttf2tfm.exe
2021/06/26  12:28           401,920 miktex-ttftotype42.exe
2020/12/14  11:08           894,976 miktex-update.exe
2020/12/14  11:08           894,976 miktex-update_admin.exe
2021/06/26  12:43           216,064 miktex-upmendex.exe
2021/06/26  12:42           208,896 miktex-vftovp.exe
2021/06/26  12:42           195,584 miktex-vptovf.exe
2021/06/26  12:42           201,728 miktex-weave.exe
2020/12/14  11:00           966,144 miktex-xdvipdfmx.exe
2020/12/14  11:07         1,005,056 miktex-xelatex.exe
2021/06/26  12:44         1,236,480 miktex-xetex.exe
2021/06/26  12:44           207,872 miktex-xml2pmx.exe
2010/09/23  14:21           356,864 miktex-zip.exe
2020/12/14  10:52           274,432 MiKTeX201200-apr.dll.(old)
2020/12/14  10:52           201,728 MiKTeX201200-aprutil.dll.(old)
2020/12/14  10:53         2,377,216 MiKTeX201200-core.dll.(old)
2020/12/14  10:52           171,520 MiKTeX201200-expat.dll.(old)
2020/12/14  10:53           151,552 MiKTeX201200-extractor.dll.(old)
2020/12/14  10:52           263,680 MiKTeX201200-fmt.dll.(old)
2020/12/14  10:52         1,994,752 MiKTeX201200-libressl-crypto.dll.(old)
2020/12/14  10:53         3,044,352 MiKTeX201200-log4cxx.dll.(old)
2020/12/14  10:52           214,528 MiKTeX201200-lzma2.dll.(old)
2020/12/14  10:52            38,400 MiKTeX201200-md5.dll.(old)
2020/12/14  10:54         1,308,672 MiKTeX201200-packagemanager.dll.(old)
2020/12/14  10:55            78,848 MiKTeX201200-popt.dll.(old)
2020/12/14  10:54           785,408 MiKTeX201200-setup.dll.(old)
2020/12/14  10:52           121,344 MiKTeX201200-trace.dll.(old)
2020/12/14  10:54           406,528 MiKTeX201200-ui-Qt.dll.(old)
2020/12/14  10:52            93,184 MiKTeX201200-utf8wrap.dll.(old)
2020/12/14  10:52           139,264 MiKTeX201200-util.dll.(old)
2021/06/26  11:39           298,496 MiKTeX210628-app.dll
2021/06/26  11:19           275,456 MiKTeX210628-apr.dll
2021/06/26  11:20           202,752 MiKTeX210628-aprutil.dll
2021/06/26  11:21           104,448 MiKTeX210628-bzip2.dll
2021/06/26  11:40         1,212,416 MiKTeX210628-cairo.dll
2021/06/26  12:14            42,496 MiKTeX210628-core-PS.dll
2021/06/27  10:58         2,417,664 MiKTeX210628-core.dll
2021/06/26  11:32           567,296 MiKTeX210628-curl.dll
2021/06/26  12:58            80,384 MiKTeX210628-dib.dll
2021/06/26  13:00           932,864 MiKTeX210628-dvi.dll
2021/06/26  11:20           175,616 MiKTeX210628-expat.dll
2021/06/26  11:32           146,432 MiKTeX210628-extractor.dll
2021/06/26  11:18           265,728 MiKTeX210628-fmt.dll
2021/06/26  11:39           375,808 MiKTeX210628-fontconfig.dll
2021/06/26  11:29           356,864 MiKTeX210628-freeglut.dll
2021/06/26  11:30           816,640 MiKTeX210628-freetype2.dll
2021/06/26  11:43           506,368 MiKTeX210628-gd.dll
2021/06/26  11:34            40,960 MiKTeX210628-getopt.dll
2021/06/26  11:44           236,032 MiKTeX210628-graphite2.dll
2021/06/26  11:46         3,870,208 MiKTeX210628-harfbuzz.dll
2021/06/26  11:49           693,760 MiKTeX210628-hunspell.dll
2021/06/26  11:43           360,448 MiKTeX210628-jpeg.dll
2021/06/26  12:12           155,136 MiKTeX210628-kpathsea.dll
2021/06/26  11:24         1,996,800 MiKTeX210628-libressl-crypto.dll
2021/06/26  11:33           752,128 MiKTeX210628-loc.dll
2021/06/26  11:39         1,915,904 MiKTeX210628-log4cxx.dll
2021/06/26  12:28           284,160 MiKTeX210628-lua53.dll
2021/06/26  11:21           216,064 MiKTeX210628-lzma2.dll
2021/06/26  11:21            39,936 MiKTeX210628-md5.dll
2021/06/26  12:29           916,480 MiKTeX210628-metapost.dll
2021/06/26  11:53           515,584 MiKTeX210628-mpfr.dll
2021/06/26  11:32           116,736 MiKTeX210628-mspack.dll
2021/06/26  12:34            47,104 MiKTeX210628-packagemanager-PS.dll
2021/06/26  11:33         1,318,912 MiKTeX210628-packagemanager.dll
2021/06/26  11:30           885,760 MiKTeX210628-pixman.dll
2021/06/26  11:30           258,560 MiKTeX210628-png.dll
2021/06/26  12:00           845,312 MiKTeX210628-poppler-Qt.dll
2021/06/26  11:59         5,189,120 MiKTeX210628-poppler.dll
2021/06/26  12:05            80,896 MiKTeX210628-popt.dll
2021/06/26  12:13           100,352 MiKTeX210628-regex.dll
2021/06/26  11:33            58,880 MiKTeX210628-res.dll
2021/06/26  11:34           802,304 MiKTeX210628-setup.dll
2021/06/26  12:40           274,944 MiKTeX210628-teckit.dll
2021/06/26  12:12           531,456 MiKTeX210628-texmf.dll
2021/06/26  11:21           121,856 MiKTeX210628-trace.dll
2021/06/26  11:35           462,336 MiKTeX210628-ui-Qt.dll
2021/06/26  11:29            69,632 MiKTeX210628-unxemu.dll
2021/06/26  11:24           126,976 MiKTeX210628-uriparser.dll
2021/06/26  11:18            95,232 MiKTeX210628-utf8wrap.dll
2021/06/26  11:18           299,520 MiKTeX210628-util.dll
2021/06/26  12:14           114,688 MiKTeX210628-web2c.dll
2021/06/26  11:18           127,488 MiKTeX210628-zlib.dll
2021/06/26  12:10            56,832 MiKTeX210628-zzip.dll
2021/06/26  12:34           236,544 miktexsetup.exe
2020/12/14  11:05            63,488 mk4ht.exe
2020/12/14  11:11         6,423,552 mkcharacters.exe
2021/06/26  12:34           296,960 mkfntmap.exe
2020/12/14  11:11         6,423,552 mkglyphlist.exe
2020/12/14  11:05            63,488 mkjobtexmf.exe
2020/12/14  11:11         6,423,552 mkluatexfontdb.exe
2020/12/14  11:03            72,192 mkocp.exe
2020/12/14  11:05            63,488 mkt1font.exe
2020/12/14  11:00           566,272 mktexlsr.exe
2020/12/14  11:05         2,630,656 mllatex.exe
2020/12/14  11:05         2,630,656 mltex.exe
2020/12/14  11:00           303,104 mmafm.exe
2020/12/14  11:00           379,392 mmpfb.exe
2021/06/26  12:34           318,464 mpm.exe
2020/12/14  11:03           162,304 mpost.exe
2020/12/14  11:05            63,488 mptopdf.exe
2020/06/09  09:32           590,112 msvcp140.dll
2020/06/09  09:32            31,728 msvcp140_1.dll
2020/06/09  09:32           193,520 msvcp140_2.dll
2020/06/09  09:32            27,416 msvcp140_codecvt_ids.dll
2021/06/26  12:35           172,032 mthelp.exe
2021/06/26  13:03           197,120 mtprint.exe
2020/12/14  11:11         6,423,552 musixflx.exe
2020/12/14  11:11         6,423,552 musixtex.exe
2020/12/14  11:03           202,752 odvicopy.exe
2020/12/14  11:03           215,552 ofm2opl.exe
2020/12/14  11:05            63,488 OOopict.exe
2020/12/14  11:03           228,864 opl2ofm.exe
2020/12/14  11:11         6,413,312 optex.exe
2021/06/26  12:39            65,536 optexcount.exe
2020/12/14  11:05            63,488 orderrefs.exe
2020/12/14  11:05            63,488 ot2kpx.exe
2020/12/14  11:00           573,952 otfinfo.exe
2020/12/14  11:00           885,248 otftotfm.exe
2020/12/14  11:03            92,160 otp2ocp.exe
2020/12/14  11:03            72,192 outocp.exe
2020/12/14  11:03           246,272 ovf2ovp.exe
2020/12/14  11:03           250,368 ovp2ovf.exe
2020/12/14  11:03           164,864 patgen.exe
2020/12/14  11:05            63,488 pdf2dsc.exe
2020/12/14  11:05            63,488 pdf2ps.exe
2020/12/14  11:05            63,488 pdfatfi.exe
2020/12/14  10:55            75,264 pdfattach.exe
2020/12/14  11:05            63,488 pdfbook2.exe
2020/12/14  11:07           236,544 pdfclose.exe
2020/12/14  11:05            63,488 pdfcrop.exe
2020/12/14  11:05         2,630,656 pdfcslatex.exe
2020/12/14  11:05         2,630,656 pdfcsplain.exe
2020/12/14  11:07           236,544 pdfdde.exe
2020/12/14  10:55            81,408 pdfdetach.exe
2020/12/14  11:05         2,630,656 pdfetex.exe
2020/12/14  10:55            81,408 pdffonts.exe
2020/12/14  10:55            96,768 pdfimages.exe
2020/12/14  10:55           146,432 pdfinfo.exe
2020/12/14  11:05         2,630,656 pdfjadetex.exe
2020/12/14  11:05         2,630,656 pdflatex-dev.exe
2020/12/14  11:05         2,630,656 pdflatex.exe
2020/12/14  11:05            63,488 pdflatexdef.exe
2020/12/14  11:05            63,488 pdflatexpicscale.exe
2020/12/14  11:07           236,544 pdfopen.exe
2020/12/14  11:05            63,488 pdfopt.exe
2020/12/14  10:55            75,776 pdfseparate.exe
2020/12/14  11:05         2,630,656 pdftex.exe
2020/12/14  11:05            63,488 pdftexdef.exe
2020/12/14  10:55           217,088 pdftocairo.exe
2020/12/14  10:55           203,264 pdftohtml.exe
2020/12/14  10:55            90,624 pdftoppm.exe
2020/12/14  10:55            90,112 pdftops.exe
2020/12/14  11:04         1,525,760 pdftosrc.exe
2020/12/14  10:55           123,904 pdftotext.exe
2020/12/14  10:55           102,912 pdfunite.exe
2020/12/14  11:05         2,630,656 pdfxmltex.exe
2020/12/14  11:05            63,488 pedigree.exe
2020/12/14  11:05            63,488 perltex.exe
2020/12/14  11:05            63,488 pf2afm.exe
2020/12/14  11:11         6,423,552 pfarrei.exe
2020/12/14  11:05            63,488 pfbtopfa.exe
2020/12/14  11:05            63,488 pftogsf.exe
2020/12/14  11:05            63,488 pgfplots.exe
2020/12/14  11:05            74,240 pk2bm.exe
2020/12/14  11:05            63,488 pkfix-helper.exe
2020/12/14  11:05            63,488 pkfix.exe
2021/07/09  19:06    <DIR>          platforms
2020/12/14  11:06           175,616 pltotf.exe
2020/12/14  11:05           645,632 pmxab.exe
2020/12/14  11:05            63,488 pn2pdf.exe
2020/12/14  11:05           124,928 pooltype.exe
2020/12/14  11:01           192,512 prepmx.exe
2020/12/14  11:05            63,488 ps2ascii.exe
2020/12/14  11:05            63,488 ps2epsi.exe
2020/12/14  11:05            63,488 ps2pdf.exe
2020/12/14  11:05            63,488 ps2pdf12.exe
2020/12/14  11:05            63,488 ps2pdf13.exe
2020/12/14  11:05            63,488 ps2pdf14.exe
2020/12/14  11:05            63,488 ps2pdfxx.exe
2020/12/14  11:05           210,944 ps2pk.exe
2020/12/14  11:05            63,488 ps2ps.exe
2020/12/14  11:05            63,488 ps2ps2.exe
2020/12/14  11:05            63,488 ps4pdf.exe
2020/12/14  11:05            78,848 psbook.exe
2020/12/14  11:05            63,488 psjoin.exe
2020/12/14  11:05            89,088 psnup.exe
2020/12/14  11:05            87,552 psresize.exe
2020/12/14  11:05            79,872 psselect.exe
2020/12/14  11:05            63,488 pst2pdf.exe
2020/12/14  11:05            88,576 pstops.exe
2020/12/14  11:11         6,423,552 ptex2pdf.exe
2020/12/14  11:05            63,488 pygmentex.exe
2020/12/14  11:05            63,488 pypostproc.exe
2020/12/14  11:05            63,488 pypreproc.exe
2020/12/14  11:05            63,488 pythontex.exe
2020/11/06  07:29            33,264 Qt5Concurrent.dll
2020/11/06  07:29         6,023,664 Qt5Core.dll
2020/05/11  10:46         5,998,712 Qt5Core.dll.(old)
2020/11/06  07:29         7,008,240 Qt5Gui.dll
2020/05/11  10:47         7,085,176 Qt5Gui.dll.(old)
2020/11/06  07:29         1,340,400 Qt5Network.dll
2020/11/06  07:29           317,424 Qt5PrintSupport.dll
2020/11/06  10:41         3,591,664 Qt5Qml.dll
2020/11/06  11:19         1,243,120 Qt5Script.dll
2020/11/06  11:19           569,328 Qt5ScriptTools.dll
2020/11/06  07:30         5,498,352 Qt5Widgets.dll
2020/05/11  10:47         5,516,920 Qt5Widgets.dll.(old)
2020/11/06  11:21           237,040 Qt5WinExtras.dll
2020/05/11  17:53           469,624 Qt5WinExtras.dll.(old)
2020/11/06  07:30           213,488 Qt5Xml.dll
2020/12/14  11:05            63,488 rcsinfo.exe
2020/12/14  10:57            46,080 rebar.exe
2021/06/26  12:25           462,848 repstopdf.exe
2020/12/14  11:05            63,488 rubikrotation.exe
2020/12/14  11:05            63,488 rungs.exe
2020/12/14  11:05           124,416 scor2prt.exe
2020/12/14  10:58            36,864 sjisconv.exe
2020/12/14  10:58           114,688 sjislatex.exe
2020/12/14  10:58           114,688 sjispdflatex.exe
2020/12/14  11:11         6,423,552 songidx.exe
2020/12/14  11:05            63,488 spix.exe
2020/12/14  11:05            63,488 splitindex.exe
2020/12/14  11:05            63,488 srcredact.exe
2020/12/14  11:05            63,488 sty2dtx.exe
2021/07/09  19:06    <DIR>          styles
2020/12/14  11:05            63,488 svn-multi.exe
2020/12/14  11:05           208,896 synctex.exe
2020/12/14  11:00           376,832 t1dotlessj.exe
2020/12/14  11:00           328,704 t1lint.exe
2020/12/14  11:00           333,824 t1rawafm.exe
2020/12/14  11:00           369,152 t1reencode.exe
2020/12/14  11:00           349,184 t1testpage.exe
2020/12/14  11:05            98,304 t4ht.exe
2020/12/14  10:57           168,960 tangle.exe
2021/06/26  12:40           257,536 tdsutil.exe
2020/12/14  11:06           600,576 tex.exe
2020/12/14  10:57            46,592 tex2aspc.exe
2020/12/14  11:11         6,423,552 tex4ebook.exe
2020/12/14  11:05           174,592 tex4ht.exe
2020/12/14  11:05            63,488 texcount.exe
2020/12/14  11:05            63,488 texdef.exe
2020/12/14  11:05            63,488 texdiff.exe
2020/12/14  11:05            63,488 texdirflatten.exe
2020/12/14  11:03           165,376 texdoc.exe
2020/12/14  11:05            63,488 texfot.exe
2020/12/14  11:00           566,272 texhash.exe
2021/06/26  12:41           271,872 texify.exe
2020/12/14  11:05            63,488 texindy.exe
2020/12/14  11:00           566,272 texlinks.exe
2020/12/14  11:05            63,488 texloganalyser.exe
2020/12/14  11:11         6,413,312 texlua.exe
2020/12/14  11:11         6,413,312 texluac.exe
2020/12/14  11:05            63,488 texosquery-jre5.exe
2020/12/14  11:05            63,488 texosquery-jre8.exe
2020/12/14  11:05            63,488 texosquery.exe
2020/12/14  11:05            63,488 texplate.exe
2020/12/14  11:07         7,362,560 texworks.exe
2020/12/14  11:06           178,176 tftopl.exe
2020/12/14  11:05            63,488 thumbpdf.exe
2020/12/14  11:04           102,912 ttf2afm.exe
2020/12/14  11:05            63,488 ttf2kotexfont.exe
2020/12/14  11:06           156,672 ttf2pk.exe
2020/12/14  11:06           180,736 ttf2tfm.exe
2020/12/14  11:00           398,848 ttftotype42.exe
2020/12/14  11:05            63,488 txt2qq.exe
2019/03/18  19:50         1,011,656 ucrtbase.dll
2020/12/14  11:02           287,744 updmap.exe
2020/12/14  11:06           213,504 upmendex.exe
2020/12/14  11:05            63,488 urlbst.exe
2020/12/14  11:05            63,488 utf8-to-dad.exe
2020/12/14  11:05         2,630,656 utf8mex.exe
2020/06/09  09:32           337,392 vccorlib140.dll
2020/06/09  09:32           101,872 vcruntime140.dll
2020/06/09  09:32            44,528 vcruntime140_1.dll
2020/12/14  11:06           206,848 vftovp.exe
2020/12/14  11:02           718,336 virmf.exe
2020/12/14  11:06           600,576 virtex.exe
2020/12/14  11:05            63,488 vpe.exe
2020/12/14  11:05            63,488 vpl2ovp.exe
2020/12/14  11:05            63,488 vpl2vpl.exe
2020/12/14  11:06           192,512 vptovf.exe
2020/12/14  11:06           199,680 weave.exe
2020/12/14  11:05            63,488 webquiz.exe
2020/12/14  11:05            63,488 wmakebat.exe
2020/12/14  11:00           966,144 xbb.exe
2020/12/14  11:05            63,488 xdv2pdf_mergemarks.exe
2020/12/14  11:00           966,144 xdvipdfmx.exe
2020/12/14  11:07         1,005,056 xelatex-dev.exe
2020/12/14  11:07         1,005,056 xelatex.exe
2020/12/14  11:05            63,488 xelatexdef.exe
2020/12/14  11:07         1,005,056 xetex.exe
2020/12/14  11:05            63,488 xetexdef.exe
2020/12/14  11:11         6,423,552 xindex.exe
2020/12/14  11:05            63,488 xindy.exe
2021/06/26  12:44           207,872 xml2pmx.exe
2020/12/14  11:05         2,630,656 xmltex.exe
2021/06/26  13:02         1,542,144 yap.exe
2010/09/23  14:21           356,864 zip.exe
             696 File(s)    656,332,744 bytes
```
</details>

I executed `initexmf --admin --mkmaps --mklinks` since this was the aborted command during the update (at least the error I could see in the logs), but this doesn't cause the errors to disappear. Probably there are more commands to be executed in the regular process which I am not aware of.

**Comment by edocevoli** [comment]
Please try

```
initexmf --admin --mkmaps --mklinks --force
```

**Comment by DigNative** [comment]
Executing the command

> ```
> initexmf --admin --mkmaps --mklinks --force
> ```

as proposed by you worked and the installation is back to normal now. I cannot observe an issue anymore. 😀 I have a workaround now which a can apply to our systems. Thank you very much for your fast and very good support!

In parallel, I verified that this issue is actually reproducible on vanilla Windows systems (not integrated into our corporate systems, to exclude the possibility that our particular configuration of Windows/operational environment is contributing to the issue): If I repeat the steps from my original post using an MiKTeX installation in the repository state as of November 2020 on a newly installed vanilla Windows 10 VM, I observe the same issue after upgrading. So it will potentially affect many users with older MiKTeX installations trying to update at a particular point in time.

**Comment by DigNative** [comment]
I can confirm that this issue still exists and also appears during new installations. As a workaround, we are currently manually adding a `[SHARED_MIKTEX_INST_PATH]\miktex\config\miktex.ini` with the contents mentioned in https://github.com/MiKTeX/miktex/issues/868#issuecomment-877343381 above before starting the MiKTeX installation using the `miktexsetup_standalone` installer.

If we don't do this, the installation fails on all of our computers, tested with different versions of Windows 10.
